docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Best practices: Analyzing shop data"
+description: Learn how to analyze and present shop data.
+last_updated: Oct 24, 2024
+template: back-office-user-guide-template
+related:
+---
+
+Amazon Quicksight is a powerful business intelligence tool. It's highly customizable and has a lot of tools that can't be covered in one document. This document describes a typical use case and how you can start working with analytics, but doesn't cover all the editing options.
+
+For more insights into creating analyses and dashboards, see [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).
+
+## Prerequisites
+
+[Set up Amazon QuickSight](/docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.html)
+
+
+## Optional: Back up the default analysis
+
+The Analytics feature is shipped with a default analysis that can be useful while you're learning how to create your own. If you need to play around with the existing analysis, we recommend duplicating the default one as follows:
+
+1. In the Back Office, go to **Analytics**.
+2. On the **Analytics** page, click **Analyses**.
+3. Click on **MASTER Analysis**.
+4. On the page of the analysis, click **File**>**Save as Analysis**.
+5. In the **Save a copy** window, enter an **Analysis name**.
+6. Click **SAVE**.
+  This duplicates the analysis and opens it. Now you can safely play around with this duplicate or the original analysis.  
+
+
+
+## Refresh data
+
+In this example, you're going to create an analysis with product availability data. Because availability is constantly changing, it's important for data to be up to date.
+
+To refresh product related datasets, do the following:
+1. On the **Analytics** page, go to **Datasets**.
+2. In the **Datasets** section, click on the **Product Concrete Availability** dataset.
+  This opens the dataset's details.
+3. Click the **Refresh** tab.
+4. Click **REFRESH NOW**.
+5. In the **Refresh type** window, select **Full refresh**.
+6. In the **Confirm refresh** window, click **REFRESH**
+  This refreshes the page. A new refresh entry is displayed in the **History** pane. You can check the status of the refresh in the **Status** column.
+7. To go back to the list of datasets, cilck **< Datasets**.
+8. Repeat steps 2-7 for all the other product datasets, like **Product Concrete + Store**
```

Comment:
this one looks scary like the user has to go to 2-7 different datasets and manually update them.